src/pages/CurrentLocation.tsx

```diff
@@ -0,0 +1,46 @@
+import { Stack, Autocomplete, TextField, Box, Typography } from '@mui/material';
+import { useState } from 'react';
+
+
+const dummyResults = ['HTML', 'CSS', 'React']; // simulate ui to show results in autocomplete :
+
+type DummyItem = {
+    id: number,
+    label: string
+}
+
+const skillsOptions = dummyResults.map((dummy, index) => ({
+    id: index + 1,
+    label: dummy
+}))
+
+const CurrentLocation = () => {
+
+    const [value, setValue] = useState<DummyItem | null>(null);
```

Comment:
pretty sure it will work
```suggestion
    const [value, setValue] = useState<DummyItem>(null);
```